src/otx/cli/utils/__init__.py

```diff
@@ -2,3 +2,24 @@
 # SPDX-License-Identifier: Apache-2.0
 
 """CLI Utils."""
+
+import importlib
+import inspect
+from pathlib import Path
+
+
+def get_otx_root_path() -> str:
```

Comment:
Why don't we use `Path` as is?

Reply:
Thanks! Done - https://github.com/openvinotoolkit/training_extensions/pull/2805/commits/1a254d9d1306b9389b89a5594eefa8faf43f2e38